server/src/server.ts

```diff
@@ -257,11 +271,28 @@ function parseProxy(proxyUrl) {
 }
 
 async function inspect(identifier: VersionedTextDocumentIdentifier) {
+    const guideUrl = `${deepscanServer}/docs/deepscan/vscode#token`;
+    const generateUrl = `${deepscanServer}/dashboard/#view=account-settings`;
+    if (!token) {
+        connection.console.error('DeepScan access token is required for using the DeepScan extension. Follow these steps to register a new token.\n'
+        + `    1. You can regenerate a new token in DeepScan Account Settings page(${generateUrl}).\n`
+        + `    2. Back to VS Code and open the Command Palette\n`
+        + `    3. Type in 'Configure DeepScan Access Token For VS Code'\n`
+        + `    4. Paste in your newly generated token value.\n`
+        + `Please, visit DeepScan Guide(${guideUrl}) for more information.`);
```

Comment:
```suggestion
        connection.console.error(`Failed to inspect: DeepScan access token not configured. Visit ${guideUrl} to generate a token.`);
```

그냥 guide 문서 링크하기로 하죠.
Output 창에 자세한 instruction을 기재하는 것은 아무래도 시인성이 떨어지네요.
그리고 분석 실패할때마나 보여질테니 간결한 필요가 있는 것 같고요.